### DISCUSSION TITLE
Nextra & Shadcn UI

### DISCUSSION THREAD

**Discussion by ThomasDevosInforma** [shuding/nextra#2570] [Q&A]
Hi Nextra Community,

I am thinking to switch over to Nextra for my company documentation website mainly because Nextra is built on top of NextJS but I was wondering about the best way to integrate a nextra app with Shadcn UI to extend the power of mdx filed

Any best practices are more than welcome. Thanks in advance
Thomas

**Comment by ic4-y** [comment]
Good question @ThomasDevosInforma did you find an answer? Did you end up integrating Shadcn UI?

**Comment by ThomasDevosInforma** [comment]
yes @icodeforyou-dot-net and it works pretty well. I am using Nextra 3.0 alpha.
